src/Corporation/helpers.ts

```diff
@@ -88,3 +88,11 @@ export function issueNewSharesFailureReason(corp: Corporation, numShares: number
 
   return "";
 }
+
+export function isValuationHighEnoughToBribe(valuation: number) {
```

Comment:
These two functions are so simple that I'm not sure they need to/should be made functions.

I.e. if I saw `valuation >= corpConstants.bribeThreshold` in the code that would be more clear to me than `isValuationHighEnoughToBribe(valuation)`, and it's roughly the same length.